src/bindings/python/setup.in.py

```diff
@@ -0,0 +1,49 @@
+""" libCellML Library: A library for the parsing, printing, and manipulation
+of CellML 2.0 compliant models.
+
+"""
+
+classifiers = """\
+Development Status :: 5 - Production/Stable
+Intended Audience :: Developers
+Intended Audience :: Education
+Intended Audience :: Science/Research
+License :: OSI Approved :: Apache Software License
+Programming Language :: Python
+Operating System :: Microsoft :: Windows
+Operating System :: Unix
+Operating System :: MacOS :: MacOS X
+Topic :: Software Development :: Libraries :: Python Modules
+"""
+
+from setuptools import setup
+from setuptools.dist import Distribution
+
+
+doclines = __doc__.split("\n")
+
+
+class BinaryDistribution(Distribution):
+    def is_pure(self):
+        return False
+
+    def has_ext_modules(self):
+        return True
+
+
+setup(
+    name='@PYPI_PACKAGE_NAME@',
+    version='@libCellML_VERSION@@LIBCELLML_DEVELOPER_VERSION@',
+    author='libCellML Developers',
+    author_email='libcellml@googlegroups.com',
+    packages=['libcellml'],
+    package_data={'libcellml': [@SETUP_PY_PACKAGE_FILES_STR@]},
+    url='http://cellml.org',
+    license='Apache Software License',
```

Comment:
Shouldn't this simply read `Apache 2.0` rather than `Apache Software License`?

Reply:
This comes from the [Python trove classifiers](https://pypi.org/classifiers/) we want to follow standards I think.

Reply:
Sure, but then how do we distinguish between Apache 1 and Apache 2?...

Reply:
We don't and I don't think it is something that we need to do.

Reply:
Ok, fair enough. I guess I am happy with whatever standard is followed by the Python community.